src/provider/index.ts

```diff
@@ -1,5 +1,7 @@
 import * as slack from './slack';
+import * as oauth from './oauth';
 
 export * as slack from './slack';
+export * as oauth from './oauth';
 
-export type INative = slack.INative;
+export type INative = slack.INative | any;
```

Comment:
Please stay consistent with the other code and export an `oauth.INative` type.

Reply:
done